tests/AppTest.cpp

```diff
@@ -1933,7 +1933,6 @@ TEST_CASE_METHOD(TApp, "AllowExtrasArgModify", "[app]") {
     app.allow_extras();
     app.add_option("-f", v2);
     args = {"27", "-f", "45", "-x"};
-    auto cargs = args;
```

Comment:
Unused, hopefully this was not supposed to be used later for a missing test?

Reply:
Seeing how the method names is argModify I could see that being potentially useful, but it look like the test didn't end up using it and just making the check with a constructed vector.